src/python/ksc/torch_frontend.py

```diff
@@ -578,13 +583,13 @@ def cpp_string_to_module(
 
 
 def ksc_defs_to_autograd_function(
-    ksc_defs, entry_def, torch_extension_name, elementwise=False, generate_lm=True
+    ksc_defs, entry_def, torch_extension_name, vectorization=False, generate_lm=True
```

Comment:
`vectorization=False` should be `vectorization=VecSpec_None()`? (Sorry, only noticed this when trying to merge into my branch.)